protocol/protocol.proto

```diff
@@ -252,6 +258,40 @@ message SpacePayload {
     oneof content {
         Inception inception = 1;
         ChannelUpdate channel = 2;
+        ChunkedMedia space_image = 3;
```

Comment:
I know we spec'd this property out to be unencrypted, i think we need to encrypt it (it should also be encrypted with a derived key). 
I don't want unencrypted thumbnails in the space stream.

Reply:
@brianathere or @sergekh2  what do you think?

Reply:
The protocol.proto has this:

```
oneof encryption {
        AESGCM aesgcm = 101;
        DerivedAESGCM derived = 102;
    }
```
We can apply the encryption algo to the thumbnail as well.
Also, I have a question about the original definition of AESGCM:

```
 message AESGCM {
        bytes iv = 1;
        bytes secretKey = 2;
    }
```
What encrypts this?

Reply:
> @brianathere or @sergekh2 what do you think?

We should encrypt the thumbnail as well. For all of the streams. Yes, it means you need to downlod it to decrypt it before render, but we should not have plaintext in the stream

Reply:
Tak we should encrypt the whole payload - it becomes `EncryptedData space_image = 3;`